### DISCUSSION TITLE
Does live transcription support Opus?

### DISCUSSION THREAD

**Discussion by robfig** [deepgram/community#467] [General help] [answered]
I have WebRTC audio packets, which are Opus format, that I want to use for live/streaming transcription. The documentation suggests that the API supports this audio format, but I couldn't find an explicit list anywhere and I also couldn't get it to work in practice, either passing the audio frames directly or by wrapping them in an OGG container. 

Can anyone confirm whether Opus is supported?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by STSonyThomas** [reply]
Could you provide an example code where opus is used.

**Comment by lukeocodes** [comment, marked as answer]
Yes, Deepgram supports Opus 😊

**Reply by STSonyThomas** [reply]
Hey I don't see an example where OPUS format is used for livestream transcription.
